ruff.toml

```diff
@@ -0,0 +1,4 @@
+extend-include = ["*.ipynb"]
```

Comment:
```suggestion
extend-include = ["*.ipynb"]
extend-exclude = ["__init__.py"]

[lint]
ignore = [
  "E741", 
]
```

Reply:
have check prorper if dont add `types_or: [python, pyi, jupyter]` this to pre-commit-config.yaml then it is not checking .ipynb

i mean if i remove `"E703"` from ruff.toml and then try to run the test then there is no changes in .ipynb it did not remove `;`.

Reply:
Yes, this should be expected, since we don't want to remove the semicolons